packages/angular/cli/lib/config/schema.json

```diff
@@ -342,6 +342,9 @@
           "type": "string",
           "description": "Root of the project files."
         },
+        "i18n": {
```

Comment:
Note: this option should only be application to "application" projects, though didn't manage to do a schema validation for this yet.